applications/Archisinal.md

```diff
@@ -0,0 +1,310 @@
+
+## Development Roadmap :nut_and_bolt: ##
+
+### Overview
+
+The development roadmap for Archisinal includes multiple milestones to ensure the successful launch and operation of the platform.
+
+### **Overview**
+
+- **Total Estimated Duration:**
+    - MVP: 2.5 months
+    - Post-MVP: 9 months
+    - Post-Launch: 10 months
+- **Full-Time Equivalent (FTE): 6**
+- **Total Costs:**
+    - MVP: $60,000
+
+## MVP (brief discussion)
+
+### **Milestone 1: Smart Contract Development**
+
+- **Estimated duration:** 1 month
+- **FTE:** 3
+- **Costs:** 24,000 USD
+
+**Overview:**
+
+- Develop smart contracts for NFT minting, buying, selling, and royalty distribution
+- Testing and deployment on Astar testnet Shibuya
+
+| Number | Deliverable | Specification |
+| --- | --- | --- |
+| 0a. | License | MIT |
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how users can interact with the developed smart contracts. |
+| 0c. | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit & integration tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an article that explains the development and features of the smart contracts, as well as any challenges faced and lessons learned. |
+| 1. | NFT Minting | Develop a smart contract functionality for NFT minting when creators upload their designs. |
+| 2. | NFT Transfer | Develop a smart contract functionality for NFT transfer when a sale happens, including royalty distribution. |
+| 3. | Creator’s Administering | Develop a smart contract functionality for creators, including managing their project NFTs data and tracking sales. |
+| 4. | Buyer’s Administering | Develop a smart contract functionality for buyers, including tracking their purchases and NFT holdings. |
+| 5. | Marketplace Administration | Develop a smart contract functionality for marketplace administrators, including the ability to delist inappropriate items. |
+| 6. | Auction | Develop smart contract functionality for auctioning NFTs, allowing users to place bids and manage their auctions. |
+| 7. | Test Network Deployment | Deploy the developed smart contracts on a test network (Shibuya) for evaluation and testing. |
+| 8. | Quality, Performance, and Security Considerations | Ensure that the developed smart contracts meet quality, performance (weight), and security requirements. |
+
+### Milestone 2: Basic UI/UX and production deployment
+
+- **Estimated duration:** 1.5 months
+- **FTE:** 3
+- **Costs:** 36,000 USD
+
+**Overview:**
+
+- Design and develop a user-friendly interface for the NFT marketplace. Technical stack: React + Next.js
+- Integrate wallet connectivity and support for multiple tokens (ETH, DOT, and USDT)
```

Comment:
From earlier you mentioned that you wanted to also integrate `ASTR`, would this still be the case?

Reply:
Hi @HCastano - yes it would.